aspnetcore/test/integration-tests.md

```diff
@@ -311,6 +311,10 @@ Add the *xunit.runner.json* file to root of the test project with the following
 }
 ```
 
+## Disposal of objects
+
+After the tests of the `IClassFixture` implementation are executed, [TestServer](/dotnet/api/microsoft.aspnetcore.testhost.testserver) and [HttpClient](/dotnet/api/system.net.http.httpclient) are disposed when xUnit disposes of the [WebApplicationFactory](/dotnet/api/microsoft.aspnetcore.mvc.testing.webapplicationfactory-1). If objects instantiated by the developer require disposal, dispose of them in the `IClassFixture` implementation. For more information, see [Implementing a Dispose method](/dotnet/standard/garbage-collection/implementing-dispose).
```

Comment:
Use xref links for the APIs.

Reply:
Later tho ...... the *whole topic* is full of old skool links. I'll be back here on a few more upcoming issues soon. Shortly, I'll knock them all out in one go.